client/main.lua

```diff
@@ -234,6 +234,10 @@ RegisterNetEvent('vehiclekeys:client:SetOwner', function(plate)
 end)
 -- Backwards Compatibility ONLY -- Remove at some point --
 
+RegisterNetEvent('vehiclekeys:client:SetOwnerByNetworkId', function(vehicleId)
+    TriggerServerEvent('qb-vehiclekeys:server:AcquireVehicleKeys', QBCore.Functions.GetPlate(NetworkGetEntityFromNetworkId(vehicleId)))
+end)
```

Comment:
Could we do a server side export instead, or does it have to go through the client?

Reply:
This need to pass through client that was all the problem since you couldn´t get the plate server side.

Reply:
I tried testing this yesterday on a server and some people weren't getting the keys, I am guessing due to connection issues? Would a Wait help here?

Reply:
Well some times you dont wrap on car and then they would not receive the keys. But thats not a problem on this event but a problem in the CreateVehicle function of qbx-core, im will try to find a solution :)

Reply:
I didn't catch how they did it, I tried testing locally and it worked fine on me, not sure if it was due to cache that I was getting the keys? I mean the CreateVehicle function has the warp but the triggerclientevent is after, it should still run it?

Reply:
This is the one used by qbx-core in the GetPlate function but its not working, i have tried it (isolated dont using the GetPlate of qbx-core) and its literally dont return none, i think its because of the CreateVehicleServerSetter, you could try by yourself

Reply:
> I didn't catch how they did it, I tried testing locally and it worked fine on me, not sure if it was due to cache that I was getting the keys? I mean the CreateVehicle function has the warp but the triggerclientevent is after, it should still run it?

No sorry, i express myself wrong, in the case of garage it dosent even wrap, what i am saying is that im getting the same problem and when it happens and i have the wrap as true the ped is not wrapped, the problem here i think is cause the vehicle was not spawned yet and they are already using the veh var to wrap ped into car or to send keys but it is not defined yet. Maybe a solution for that is adding:

while (not veh) do Wait(0) end

After vehicle creation to avoid any trouble caused by some connection issue. I need to test it